addon/components/x-toggle/template.hbs

```diff
@@ -1,19 +1,26 @@
-{{#if showLabels}}
-	<span class="toggle-text toggle-prefix {{size}}">
-		{{offLabelValue}}
-	</span>
-{{/if}}
+<span class="x-toggle-component">
+  {{#if showLabels}}
+  	<span class="toggle-text toggle-prefix off-label" onclick={{action 'setToValue' false}}>
+  		{{_offLabel}}
+  	</span>
+  {{/if}}
+  <span
+    id={{elementId}}
+    class="x-toggle-container {{size}}{{if toggled ' x-toggle-container-checked'}}{{if disabled ' x-toggle-container-disabled'}}"
+    onclick={{action 'onClick'}}
+  >
+    {{input id=forId type='checkbox' checked=toggled class='x-toggle' disabled=disabled name=name}}
```

Comment:
I'm primarily looking at Integration tests (which I've been porting over from unit tests). Hope that's ok with you guys, I find it to be a better test of a component in general. 

Within this batch of tests, the one that is baffling me is that the `_onValue` and `_offValue` properties in the the test "changing container's value changes toggled state" are coming as the capitalized _labels_ not the intended lowercased _values_. Parsing the input of `onLabelValue` / `offLabelValue` works fine in the demo app but in this test it fails which throws off the whole test.